gateway-ha/src/main/java/io/trino/gateway/ha/security/LbLdapClient.java

```diff
@@ -76,6 +76,7 @@ public LbLdapClient(LdapConfiguration ldapConfig) {
     poolConfig.setMaxIdle(1);
     poolConfig.setMaxTotal(1);
```

Comment:
Btw, how about increasing number of connections to idk default value of 8? 
Any ideas on why it is set as 1?

Reply:
Sure, changed it so that it reads Authentication ldap conf. PTAL